src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,112 @@
+package duke;
+
+import java.io.FileNotFoundException;
+
+/**
+ * Parser deals with users' commands, differentiates each type of task.
+ */
+public class Parser {
+    /**
+     * Command enum lists out all the types of commands.
+     */
+    public enum Command {
+        BYE("bye"), LIST("list"), DONE("done"), TODO("todo"), DEADLINE("deadline"),
+              EVENT("event"), DELETE("delete"), FILTER("filter"), FIND("find");
```

Comment:
I like the use of enum here, resulting in neat code when parsing user's input.

Reply:
agree with the comment above, makes code a lot neater